### ISSUE TITLE
[1.12.2] Wireless Universal Terminal game crash when equipped as bauble

### ISSUE THREAD

**Issue by jdkleuver** [ExtraCells/ExtraCells2#641]
Crash Log: https://pastebin.com/SSZx3ScE

Problem seems to be that the bauble render method has not been implemented (is abstract) 

extracells/item/ItemWirelessTerminalUniversal.scala line 242 I believe is the problem line

**Comment by AxelMontini** [comment]
Can reproduce (just happened to me as well)

**Comment by BoltAxion** [comment]
Hey, ive just got the exact same thing. 
wanted to try the Universal terminal since i was working on a fluid network etc etc.
then wanted to Shift+click the universal terminal while i was in my me2 wireless terminal and it crashes the game. restarted a few times.  
 
Can walk, can fly, Press E = crash. 
so baubles and extracells2 dont like each other :P

PasteBin: https://pastebin.com/4JE7QMGF

**Comment by BrockWS** [comment]
Duplicate of #619

**Comment by FoxMcloud5655** [comment]
This is still happening...  Crash method below:

```
java.lang.AbstractMethodError: Method extracells/item/ItemWirelessTerminalUniversal$.getRender()Lp455w0rd/ae2wtlib/api/client/IBaubleRender; is abstract
	at extracells.item.ItemWirelessTerminalUniversal$.getRender(ItemWirelessTerminalUniversal.scala)
	at p455w0rd.ae2wtlib.client.render.BaubleRenderDispatcher.doRenderLayer(BaubleRenderDispatcher.java:43)
	at p455w0rd.ae2wtlib.client.render.BaubleRenderDispatcher.func_177141_a(BaubleRenderDispatcher.java:19)
	at net.minecraft.client.renderer.entity.RenderLivingBase.func_177093_a(RenderLivingBase.java:439)
	at net.minecraft.client.renderer.entity.RenderLivingBase.func_76986_a(RenderLivingBase.java:186)
	at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:71)
	at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:28)
	at net.minecraft.client.renderer.entity.RenderManager.func_188391_a(RenderManager.java:384)
	at net.minecraft.client.gui.inventory.GuiInventory.func_147046_a(SourceFile:128)
	at baubles.client.gui.GuiPlayerExpanded.func_146976_a(GuiPlayerExpanded.java:100)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:75)
	at net.minecraft.client.renderer.InventoryEffectRenderer.func_73863_a(InventoryEffectRenderer.java:51)
	at baubles.client.gui.GuiPlayerExpanded.func_73863_a(GuiPlayerExpanded.java:78)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1124)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1119)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by scrouthtv** [comment]
Can reproduce, happened to me as well. The game crashed for every client on the server.

`The game crashed whilst rendering entity in world
Error: java.lang.AbstractMethodError: Method extracells/item/ItemWirelessTerminalUniversal$.getRender()Lp455w0rd/ae2wtlib/api/client/IBaubleRender; is abstract`

It literally says in the tooltip can be equipeed as Bauble?

**Comment by Rurido** [comment]
Can reproduce; happened to me as well. (Modpack "A Bit Of Everything 3.3.9" for Minecraft 1.12.2)
The client crashes whenever I join the server and open up the inventory.
``` 
Description: Rendering entity in world

java.lang.AbstractMethodError: Method extracells/item/ItemWirelessTerminalUniversal$.getRender()Lp455w0rd/ae2wtlib/api/client/IBaubleRender; is abstract
	at extracells.item.ItemWirelessTerminalUniversal$.getRender(ItemWirelessTerminalUniversal.scala)
	at p455w0rd.ae2wtlib.client.render.BaubleRenderDispatcher.doRenderLayer(BaubleRenderDispatcher.java:43)
	at p455w0rd.ae2wtlib.client.render.BaubleRenderDispatcher.func_177141_a(BaubleRenderDispatcher.java:19)
	at net.minecraft.client.renderer.entity.RenderLivingBase.func_177093_a(RenderLivingBase.java:439)
	at net.minecraft.client.renderer.entity.RenderLivingBase.func_76986_a(RenderLivingBase.java:186)
	at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:71)
	at net.minecraft.client.renderer.entity.RenderPlayer.func_76986_a(RenderPlayer.java:28)
	at net.minecraft.client.renderer.entity.RenderManager.func_188391_a(RenderManager.java:384)
	at net.minecraft.client.gui.inventory.GuiInventory.func_147046_a(SourceFile:128)
	at net.minecraft.client.gui.inventory.GuiInventory.func_146976_a(SourceFile:94)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:75)
	at net.minecraft.client.renderer.InventoryEffectRenderer.func_73863_a(InventoryEffectRenderer.java:51)
	at net.minecraft.client.gui.inventory.GuiInventory.func_73863_a(SourceFile:75)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:396)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1124)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1119)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:398)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by TheInfamousAlk** [comment]
Same, still happening even when all relevant mods are up to date.

**Comment by TheInfamousAlk** [comment]
Has anyone found a workaround for this? Obviously don't put it in the bauble slot, but sometimes when I shift click it goes in, and when I pick up my grave stone, it automatically goes in. I've been having to edit my player data file to delete it every time it happens.

**Comment by FoxMcloud5655** [comment]
Other than using it standalone, no.  I was actually thinking about looking into this issue this weekend.

**Comment by BrockWS** [comment]
EC2 doesn't support newer versions of WCT/AE2WTLib. I just don't have the time to work on it and the mix of java/scala codebase doesn't help when finding bugs etc. Although you can use AE2WTLib and WCT/WFT/WPT/WIT from [p455w0rd](https://www.curseforge.com/members/therealp455w0rd/projects) and create a "Wireless Ultimate Terminal" so you don't need to use the UWT.

**Comment by ruifung** [comment]
Fixed in 1f78a0b5c15a64e26f2eadfb4b4255e13c6bf108
